services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/drs/DrsManagerImpl.java

```diff
@@ -175,4 +177,24 @@ private void validateIdHasVersion(final IdAndVersion idAndVersion) {
             throw new IllegalArgumentException("Object id should include version. e.g syn123.1");
         }
     }
+
+    @Override
+    public AccessUrl getAccessUrl(final Long userId, final String drsObjectId, final String accessId)
+            throws NotFoundException, DatastoreException, UnauthorizedException, IllegalArgumentException {
+        final UserInfo userInfo = this.userManager.getUserInfo(userId);
```

Comment:
ValidateArguments.required(userId, "userId");
ValidateArguments.required(drsObjectId, "drsObjectId");
ValidateArguments.required(accessId, "accessId");

and add unit tests for the nulls